crowdfunding_compassion/controllers/main.py

```diff
@@ -24,8 +24,12 @@ def my_account(self, form_id=None, **kw):
                 ("project_id.project_owner_id", "!=", partner.id),
             ]
         )
-        donations = participations.mapped("invoice_line_ids").filtered(
-            lambda l: l.state == "paid")
+        donations = request.env["account.invoice.line"].sudo().search(
+            [
+                ("crowdfunding_participant_id.partner_id", "=", partner.id),
+            ]
+        )
+        donations = donations.filtered(lambda l: l.state == "paid")
```

Comment:
```suggestion
```

Reply:
Indeed, I missed that and this will be way better !